tests_new/integration_tests/modules/catalog/conftest.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+from integration_tests.modules.catalog.queries import (
+    create_glossary,
+    delete_glossary,
+    create_term,
+    delete_term,
+    create_category,
+    delete_category,
+    list_glossary_associations,
+)
+
+from integration_tests.modules.s3_datasets.queries import update_dataset
+
+"""
+Temp glossary elements. Scope=function
+"""
+
+
+@pytest.fixture(scope='function')
+def glossary1(client1, group1):
+    glos = None
+    try:
+        glos = create_glossary(
+            client1, name='glossary1', group=group1, read_me='Glossary created for integration testing'
+        )
+        yield glos
+    finally:
+        if glos:
+            delete_glossary(client1, node_uri=glos.nodeUri)
+
+
+@pytest.fixture(scope='function')
+def category1(client1, group1, glossary1):
+    cat = None
+    try:
+        cat = create_category(
+            client1, name='category1', parent_uri=glossary1.nodeUri, read_me='Category created for integration testing'
+        )
+        yield cat
+    finally:
+        if cat:
+            delete_category(client1, node_uri=cat.nodeUri)
+
+
+@pytest.fixture(scope='function')
+def glossary_term1(client1, group1, glossary1):
+    term = None
+    try:
+        term = create_term(
+            client1, name='glos_term1', parent_uri=glossary1.nodeUri, read_me='Term created for integration testing'
+        )
+        yield term
+    finally:
+        if term:
+            delete_term(client1, node_uri=term.nodeUri)
+
+
+@pytest.fixture(scope='function')
+def category_term1(client1, group1, category1):
+    term = None
+    try:
+        term = create_term(
+            client1, name='cat_term1', parent_uri=category1.nodeUri, read_me='Term created for integration testing'
+        )
+        yield term
+    finally:
+        if term:
+            delete_term(client1, node_uri=term.nodeUri)
+
+
+"""
+Session glossary elements needed if using associations
+
+WARNING!
+Associations are applied to the S3_Datasets module
+Glossaries can only be tested if the S3_datasets module is enabled in the deployment used for testing!
+"""
+
+
+@pytest.fixture(scope='session')
+def session_glossary1(client1, group1):
```

Comment:
is there a reason to have 1 glossary to test CRUD operations (i.e. `glossary1`) and another glossary (i.e. `session_glossary1`) to test associations? Could this not all be handled with 1?

Reply:
The idea was to be able to reuse the glossary tag dataset outside of the module. But I think it is better to keep it simple for the moment